src/content/en/fundamentals/architecture/howto-components/howto-tabs.md

```diff
@@ -0,0 +1,587 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-04-26#}
+{# wf_published_on: 2017-04-06 #}
+
+# HowTo: Components – howto-tabs {: .page-title }
+
+{% include "web/_shared/contributors/ewagasperowicz.html" %}
+{% include "web/_shared/contributors/noms.html" %}
+{% include "web/_shared/contributors/robdodson.html" %}
+{% include "web/_shared/contributors/surma.html" %}
+
+<link rel="stylesheet" href="prism-solarizedlight.css">
+<link rel="stylesheet" href="main.css">
+
+<p>Tab panels are a pattern to limit visible content by separating
+it into multiple panels. Only one panel is visible at a time, while
+<em>all</em> corresponding tabs are always visible. To switch from one panel
+to another, the corresponding tab has to be selected.</p>
+<p>By either clicking or by using the arrow keys the user changes the
+selection of the active tab.</p>
+<p>If JavaScript is disabled, all panels are shown interleaved with the
```

Comment:
Maybe replace 'shown interleaved' to 'stacked'. It took me a second to realize what you meant by 'shown interleaved'. 

Also, 'the respective tabs' should be 'their respective tabs'.